### ISSUE TITLE
Details about Video-To-Text 2018 training and validation

### ISSUE THREAD

**Issue by nbravohack** [danieljf24/dual_encoding#16]
Greetings,

I was reading the use of the model in the Video-To-Text matching and ranking task on TRECVID 2018 ([link](https://www-nlpir.nist.gov/projects/tvpubs/tv18.papers/rucmm.pdf)) and I'm struggling to replicate the results showed in Table 2, even for run0. 

It would really help to know more details about the training process like the hyper parameters used for the model  (tuned on TRECVID 2016 VTT dataset) and how did you combine the 3 training datasets (MSVD, TGIF, MSR). What did you use for validation? 

It would help alot if I could know this information.
Thank you very much

**Comment by danieljf24** [comment]
What specific details would you like to know?

**Comment by nbravohack** [comment]
I tried to use the default configuration of the model with this hyper-parameters and I like to confirm their values or know the ones used in the Matching and Ranking Task of 2018:

Learning Rate: 0.0001
Bach Size: 128
Dropout: 0.2
Clip Normalization: 0.2
Text rrn size: 512
Video rnn size: 1024
Text kernel sizes: 2-3-4
Video kernel sizes: 2-3-4-5
Common vector space size: 2048

For the loss function I use:
Direction: "i2t"
Margin: 0.2
Max-violation: True
Cost style: sum
Measure: cosine (with previous vector normalization)

I know it is a lot of hyper parameters, but I want to be sure if I'm using the same model to discard the hyper parameter selection as the problem of not being able to replicate the results.

Thanks you so much
